js/app/restaurant/list.scss

```diff
@@ -62,15 +62,13 @@
 
   &__title {
     font-weight: 700;
-    height: ($line-height-computed * 2);
```

Comment:
Why did you remove this? It was here to make sure that all items have the same height.